### PR TITLE
font: display kps

### LINKED ISSUE #5222: front: display PK in the carto
We should be able to see the (cosmetic) field PK on the map for ponctual objects.

### PR DESCRIPTION
**Pull request by nicolaswurtz**
Close #5222 

It should display KP on all maps for: 
- detectors
- bufferstops
- signals
- psl signs (broken on angle)

Operational points kps should be displayed when backend will be updated.

### CODE REVIEW

**Comment by sim51** [comment]
Is it useful to have them at this level of zoom ? 

image
image

Should we have the PK also on the editor ?

**Review by SharglutDev — CHANGES_REQUESTED** [review]
Tested in local and it seems that even when the signals layer is disabled, the PK from the signal is still displayed.

image

**Comment by nicolaswurtz** [comment]
> Is it useful to have them at this level of zoom ?
> 
> image image
> 
> Should we have the PK also on the editor ?

Corrected !

**Comment by nicolaswurtz** [comment]
> Tested in local and it seems that even when the signals layer is disabled, the PK from the signal is still displayed.
> 
> image

corrected

**Review by SharglutDev — APPROVED** [review]
Lgtm, nice feature !

File: front/src/common/Map/Settings/MapSettingsSignals.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ import { getMap } from 'reducers/map/selectors';
 import { RootState } from 'reducers';
 
 type SettingsSignals = RootState['map']['signalsSettings'];
-const SIGNAL_SETTINGS: Array<keyof SettingsSignals> = ['all', 'stops', 'lights'];
+const SIGNAL_SETTINGS: Array<keyof SettingsSignals> = ['lights']; // 'all', 'stops' temporarely desactivated
```
Comment (sim51) [inline front/src/common/Map/Settings/MapSettingsSignals.tsx:14]: "stops" is enabled in the default state (see https://github.com/osrd-project/osrd/blob/nwz/show-pks/front/src/reducers/map/index.ts#L81C4-L81C17).
So in locally I see them, and I can't disabled them.
is it intended ?

**Comment by sim51** [comment]
You replaced the map item's label (generally the item's ID)  for Detector & BufferStop by the KP value. 

For me (as a not rails guy), it was hard to figure out that the label is the KP and not the ID/name of the object. 
Can we prefix the value by `KP `  ?

Moreover on signals, KPs are displayed just under the signal name. So there is 2 ways of how a KP is displayed on the map.

**Comment by nicolaswurtz** [comment]
> You replaced the map item's label (generally the item's ID) for Detector & BufferStop by the KP value.
> For me (as a not rails guy), it was hard to figure out that the label is the KP and not the ID/name of the object. Can we prefix the value by `KP ` ?

As the KP becomes a `label` "KP" mention is useless because it's firstly a label.

> Moreover on signals, KPs are displayed just under the signal name. So there is 2 ways of how a KP is displayed on the map.

KPs are centered under centered signals, lefted or righted along signal's position.

**Comment by sim51** [comment]
As discussed with @nicolaswurtz  : 
 * KP as label makes sense for rails expert (also without the KP prefix).
 * There is a general bug of signals, so my comment on the 'stops' is not linked to this PR

**Review by sim51 — APPROVED** [review]

**Review by Math-R — APPROVED** [review]

**Pull request merged**